raiutils/raiutils/models/models.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+
+class SKLearn(object):
+    """Provide scikit-learn related constants."""
+
+    EXAMPLES = 'examples'
+    LABELS = 'labels'
+    PREDICT = 'predict'
+    PREDICTIONS = 'predictions'
+    PREDICT_PROBA = 'predict_proba'
+
+
+def is_classifier(model):
+    return (model is not None and
```

Comment:
you may want to add some very simple test for this is_classifier method for code coverage

Reply:
These tests were written as a part of the PR https://github.com/microsoft/responsible-ai-toolbox/pull/1295